src/components/forms/JurisdictionMetadata/index.tsx

```diff
@@ -39,23 +25,9 @@ import {
 } from '../../../helpers/utils';
 import { OpenSRPService } from '../../../services/opensrp';
 
-const SUPPORTED_FORMATS = [
```

Comment:
Why did we need to remove these?

Reply:
On widows, CSV files have different MIME types depending on applications installed. I couldn't have an exhaustive list on this and so I decided just to validate by file extension instead. This Is sufficient because the user still has to download a template before uploading